bam_controls.js

```diff
@@ -253,6 +253,8 @@ class BamControls extends HTMLElement {
         this.filePicker.addEventListener('change', (event) => this.handleBedfilePick(event));
         this.removeBedFile.addEventListener("click", () => this.handleBedfileRemove());
         this.sampleMoreButton.addEventListener("click", () => this.handleSampleMoreReads());
+        // Add event listener for updating bam control input fields
+        document.addEventListener('update-bamcontrol-input', (event) => this.handleBamControlInput(event));
```

Comment:
This event is very similar to `selected-regions-change`. But we don't currently want to resample every time the user brushes, so we can't reuse that even. Let's call this one `brushed-region-change`, and change it to use the same data type as `selected-regions-change`.

You'll need to split the geneName into a separate event. Let's call that one `selected-gene-change` with the following data:

```javascript
{
  detail: {
    geneName: String(),
  }
}
```

Reply:
Updated on #111